boa_engine/Cargo.toml

```diff
@@ -42,6 +42,9 @@ console = []
 # Enable Boa's additional ECMAScript features for web browsers.
 annex-b = []
 
+# Enables the wasmbind dependency of the `chrono` crate.
+wasmbind = ["chrono/wasmbind"]
+
```

Comment:
Do we need to add a flag from our side? I think if an user wants to enable the `wasmbind` feature on `chrono`, they can add `chrono` to their `Cargo.toml` with the feature enabled.

Reply:
Yep, this is the best option. I have updated the code :)